client/assets/less/pages/places.less

```diff
@@ -1,236 +1,227 @@
-    }
-  }
-  .map-menu-buttons-wrapper {
-    .map-menu-buttons {
-      button {
-        color: @brand-dark;
-        border: 1px solid @brand-dark;
-      }
-      .dropdown-menu {
-        padding: 0;
-        border: 1px solid @brand-dark;
-        li {
-          border-bottom: 1px solid @brand-dark;
-          padding: 5px 0;
-          &:hover {
-            background-color: #f8f8f8;
-            border-radius: 4px;
+
+    .map-menu-buttons-wrapper {
+      .map-menu-buttons {
+        button {
+          color: @brand-dark;
+          border: 1px solid @brand-dark;
+        }
+        .dropdown-menu {
+          padding: 0;
+          border: 1px solid @brand-dark;
+          li {
+            border-bottom: 1px solid @brand-dark;
+            padding: 5px 0;
+            &:hover {
+              background-color: #f8f8f8;
+              border-radius: 4px;
+            }
+            &:last-child {
+              border-bottom: none;
+            }
+            a {
+              color: @brand-dark;
+              background-color: transparent;
+            }
           }
-          &:last-child {
-            border-bottom: none;
+          & filter-map-type li a {
+            display: block;
+            text-decoration: none;
+            padding: 3px 20px;
+            span {
+              float: right;
+            }
           }
-          a {
-            color: @brand-dark;
-            background-color: transparent;
+          &.place-mobile {
+            left: auto;
+            right: 0;
+            li a {
+              padding: 3px 10px;
+              span:last-child {
+                float: right;
+                &.glyphicon-refresh {
+                  animation: spinnerLittle .8s linear infinite;
+                }
+              }
+              img {
+                width: 10px;
+              }
+            }
           }
         }
-        & filter-map-type li a {
+      }
+      @media (max-width: 767px) {
+        margin-bottom: 10px;
+      }
+      @media (max-width: 348px) {
+        padding-left: 0;
+        padding-right: 0;
+      }
+
+      .dropdown-submenu {
+        position: relative;
+        & > a:after {
           display: block;
-          text-decoration: none;
-          padding: 3px 20px;
-          span {
-            float: right;
+          content: "";
+          float: right;
+          border-color: transparent;
+          border-style: solid;
+          border-width: 5px 0 5px 5px;
+          border-left-color: #ccc;
+          margin-top: 5px;
+        }
+        &:hover {
+          .dropdown-menu {
+            display: block;
+          }
+          a:after {
+            border-left-color: @brand-dark;
           }
         }
-        &.place-mobile {
-          left: auto;
-          right: 0;
-          li a {
-            padding: 3px 10px;
+        .dropdown-menu {
+          top: 0;
+          left: 100%;
+          border-radius: 0 6px 6px 6px;
```

Comment:
Here you have radius 6px but at 
.map-page-wrapper .map-menu .map-menu-buttons-wrapper .map-menu-buttons .dropdown-menu li:hover
we have border-radius 4px so  - we really need diefferent border radius?